src/platform/plugins/shared/data_views/common/types.ts

```diff
@@ -485,6 +485,11 @@ export type FieldSpec = DataViewFieldBase & {
   parentName?: string;
 
   defaultFormatter?: string;
+
+  /**
+   * Indicates whether the field is a metadata field.
+   */
+  metadata_field?: boolean;
```

Comment:
```suggestion
  isMetadataField?: boolean;
```
Is this already populated somewhere? This type is the only data views change I see in the PR. If not, can we go with a suggestion like this for consistency with the other props?

Reply:
it is populated already, see e.g. https://github.com/elastic/kibana/blob/0cc78184957fcd12110dabae50353392ea937508/src/platform/plugins/shared/data_views/server/fetcher/index_patterns_fetcher.ts#L32

this is just straightening out the types